cli/cmd/devtool/start.go

```diff
@@ -35,6 +36,10 @@ const (
 	rillGitRemote  = "https://github.com/rilldata/rill.git"
 )
 
+const (
+	minimalProvisionerSet = `{"static":{"type":"static","spec":{"runtimes":[{"host":"http://localhost:8081","slots":50,"data_dir":"dev-cloud-state","audience_url":"http://localhost:8081"}]}}}`
```

Comment:
Add a comment to explain this:
```suggestion
	// minimalProvisionerSet is used with the --minimal flag.
	// It differs from the usual dev provisioner set in not having a Clickhouse provisioner.
	minimalProvisionerSet = `{"static":{"type":"static","spec":{"runtimes":[{"host":"http://localhost:8081","slots":50,"data_dir":"dev-cloud-state","audience_url":"http://localhost:8081"}]}}}`
```